### PR TITLE
minilog and friends: elevate logSetup, refactor minilog importers

### PR DESCRIPTION
**Pull request by jasoncontrib**

### CODE REVIEW

**Comment by jcrussell** [comment]
Nice!

File: src/minilog/minilog.go [hunk 1]
```diff
@@ -33,6 +36,12 @@ const (
 )
 
 var (
+	FLogLevel = flag.String("level", "warn", "set log level: [debug, info, warn, error, fatal]")
```
Comment (jcrussell) [inline src/minilog/minilog.go]: Should be Level.

File: src/minilog/minilog.go [hunk 2]
```diff
@@ -33,6 +36,12 @@ const (
...
+	FLog      = flag.Bool("v", true, "log on stderr")
+	FLogFile  = flag.String("logfile", "", "also log to file")
```
Comment (jcrussell) [inline src/minilog/minilog.go]: Should be File.

File: src/minilog/minilog.go [hunk 3]
```diff
@@ -33,6 +36,12 @@ const (
 )
 
 var (
+	FLogLevel = flag.String("level", "warn", "set log level: [debug, info, warn, error, fatal]")
+	FLog      = flag.Bool("v", true, "log on stderr")
```
Comment (jcrussell) [inline src/minilog/minilog.go]: Should be Verbose?

File: src/minilog/minilog.go (hunk 1 above)
Comment (jcrussell) [inline src/minilog/minilog.go]: While we're at it, create a new flag type so that this can be a Level.

File: src/minilog/minilog.go [hunk 4]
```diff
@@ -132,6 +141,39 @@ func LogAll(i io.Reader, level int, name string) {
...
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+		AddLogger("file", logfile, level, false)
+	}
+}
+
 // Return the log level from a string. Useful for parsing log levels from a flag package.
 func LevelInt(l string) (int, error) {
```
Comment (jcrussell) [inline src/minilog/minilog.go]: We should copy convention in strconv.

**Comment by jcrussell** [comment]
LGTM. @djfritz -- are we ready to start 2.4 window?

**Comment by djfritz** [comment]
Rock and roll. LGTM

**Pull request merged**